Classes/Components/TemplateRenderer/StandaloneViewFactory.php

```diff
@@ -133,4 +144,38 @@ protected function setAssignedVariables(StandaloneView $view): void
             );
         }
     }
+
+    /**
+     * Returns the typoscript configuration for path : plugin.tx_dce.view
+     * @return array
+     */
+    protected function getTyposcriptViewPaths(): array
+    {
+        $views = [
+            'layoutRootPaths' => [0 => 'EXT:dce/Resources/Private/Layouts/'],
+            'templateRootPaths' => [0 => 'EXT:dce/Resources/Private/Templates/'],
+            'partialRootPaths' => [0 => 'EXT:dce/Resources/Private/Partials/'],
+        ];
+
+        if (isset($GLOBALS['TSFE']) && Compatibility::isFrontendMode()) {
```

Comment:
Hi, Thanks for your PR!
The issue I have here, is that this is limited to frontend. But the views (and those functions) are also called from backend. This might lead to unexpected behaviour for the user.

Instead, in case of backend mode, you could fetch the current page id, and build the TypoScript setup in BE. I think DCE already have got some helper functions for this... let me check

Reply:
No, no helper tools for this existing in DCE.
But this looks promising: https://stackoverflow.com/a/46050144/564000

Reply:
Thank you for your quick answer :)

We'll check that and come back when the changes and done !

Reply:
Hi Armin !

I made the changes you asked for.
As you will see, the main changes are located in the **StandaloneViewFactory** class and the **TypoScript** utility class.

The typoscript template rendering is now supported for both BE and FE relatively to the page.

Note : 
* I had to override the **BackendConfigurationManager** to be able to set the page uid of the current page.
* I replaced the injectConfigurationManager by dependency injection using the Service.yaml file

Have a great day !